src/enlighten.c

```diff
@@ -970,6 +970,23 @@ boolean dumping;
 		if(u.ucspeed==NORM_CLOCKSPEED) you_are("set to normal clockspeed");
 		if(u.ucspeed==SLOW_CLOCKSPEED) you_are("set to low clockspeed");
 		if(u.phasengn) you_are("in phase mode");
+		if (u.utemp < WARM)
+			enl_msg("Your internal boiler ", "is", "was", " under control");
+		else if (u.utemp < HOT)
+			enl_msg("Your internal boiler ", "is", "was", " running mildly warm");
+		else if (u.utemp < BURNING_HOT)
+			enl_msg("Your internal boiler ", "is", "was", " running rather hot");
+		else if (u.utemp < MELTING)
+			enl_msg("Your internal boiler ", "is", "was", " burning hot");
+		else if (u.utemp < MELTED)
+			enl_msg(Fire_resistance ? "Your thermal sinks " : "Your boiler ", "is", "was", Fire_resistance ? " nearly at capacity" : " melting to slag");
+		else 
+			enl_msg(Fire_resistance ? "Your thermal sinks " : "Your internal boiler ", "is", "was", Fire_resistance ? 
+				" are well beyond capacity, but miraculously intact" : " nothing but molten bronze");
```

Comment:
If you have fire resistance, then I think you would get a message like "Your thermal sinks **is are** well beyond capacity, but miraculously intact." The "nearly to capacity" case also has a similar issue. The cause of the issues is that "Your thermal sinks" is plural and "Your (internal) boiler" is singular. I'm not sure how to work around to make something that's grammatically correct in every case.